### ISSUE TITLE
Suggest an exisiting bike by default when there is only one bike available

### ISSUE THREAD

**Issue by OleksiyRudenko** [Gerben321/bike-care-tracker#27]
**Is your feature request related to a problem? Please describe.**
When I've got a single bike there is no much point to make a choice when there is only 1 option available.

**Describe the solution you'd like**
Set the option as selected and the UI widget disabled=true (this will actually visually denote there are no other options to pick among).
This could be possibly done not only for bikes selection but for any other drop-down lists that offer one option only, hence as a part of the UI component rendering business logic.

**Describe alternatives you've considered**
None

**Additional context**
![image](https://github.com/Gerben321/bike-care-tracker/assets/8686631/f80f691a-d6cb-4e79-b667-5c87008b122a)
![image](https://github.com/Gerben321/bike-care-tracker/assets/8686631/d7e0593b-45a1-4758-95de-c5853197952c)

**Comment by Gerben321** [comment]
That's a good one! Most people probably only have one bike so it makes sense to preselect that then. I'll have a look to see if I can do that globally indeed.

**Comment by Gerben321** [comment]
I'm fiddling around with this. There's no way to globally do this as far as I can see, but I can try to write some Javascript to apply it to all dropdowns with only one option. But this will be a bit lower priority for me because there's other way cooler things like NFC and QR codes etc. 😋

**Comment by OleksiyRudenko** [comment]
I can imagine the BE populates drop down's options before serving them to FE. This means that options generator knows how many options are available to it. So the generator could set attributes SELECTED and DISABLED to this only option.

**Comment by Gerben321** [comment]
That's right. However it's not as simple as it sounds and I would like it to be fixed in an abstract and good way. But I think I've got a good lead on how to solve it properly.
